src/google/protobuf/compiler/csharp/csharp_generator.cc

```diff
@@ -79,6 +79,8 @@ bool Generator::Generate(const FileDescriptor* file,
       cli_options.internal_access = true;
     } else if (options[i].first == "serializable") {
       cli_options.serializable = true;
+    } else if (options[i].first == "enable_nullable_reference_types") {
```

Comment:
My preference would be for a shorter option name. Microsoft just uses "nullable" (`<Nullable>` and `#nullable`) so maybe we should? Or "nullable_reference_types"?  Whatever is chosen I would be good if both the protocol buffers compiler and gRPC plugin use the same name.

Reply:
Given the requests that we've had for generating `int?`-backed properties for optional fields, I think just "nullable" could lead to confusion.

How about `enable_nrt`? I think NRT is a *reasonably* commonly-used abbrevation.

Reply:
Indeed it is too long and can be confusing, `enable_nrt` seems more appropriate